usagov/vote-gov-drupal
# Vote 2165 Accessibility dock styling

## Jira ticket
https://cm-jira.usa.gov/browse/VOTE-2165

## Description

Implement styling for the accessibility dock (all breakpoints, default and high-contrast)

## Deployment and testing

### Post-deploy steps

1. lando retune
2. cd into the theme, npm run build

### QA/Testing instructions

1. Sign in as admin
2. On the home page, confirm the styling matches the designs for desktop and mobile
3. Repeat in high-contrast mode
4. Confirm styling for Arabic (right to left language)

## Checklist for the Developer




## Checklist for the Peer Reviewers
